WalletWasabi/Services/UpdateChecker.cs

```diff
@@ -20,14 +22,33 @@ public class UpdateChecker
 
 		public WasabiClient WasabiClient { get; }
 
+		private string WorkFolderPath { get; set; }
```

Comment:
Since this is private field and is used only in the `InitializeAsync` method, it can be deleted and use the parameter `workFolderPath` directly.